search-app/docs/api.md

```diff
@@ -1961,7 +1961,12 @@ Examples of sorting by start_date in descending(Most recent data first) and asce
 
 ### <a name="retrieving-concepts-by-concept-id-and-revision-id"></a> Retrieve concept with a given concept-id or concept-id & revision-id
 
-This allows retrieving the metadata for a single concept. This is only supported for collections and granules. If no format is specified the native format of the metadata will be returned.
+This allows retrieving the metadata for a single concept. This is only supported
```

Comment:
Yikes -- you are correct. It looks like CMR's markdown parser is converting single newlines in text to `<br>`s. This is atypical ... I think you have to go out of your way to cause that behaviour in markdown parsers ... which leads me to think that if I change the configuration for CMR's markdown parsing, it will have a host of other side-effects ...